selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -2211,4 +2216,15 @@ public void launchCommentCodeFeature() {
         .keyUp(CONTROL)
         .perform();
   }
+
+  /**
+   * wait renaming field in the Editor (usually it field is used by language servers), type new
```

Comment:
What does `Language Server Field` mean? Did you mean `Code Assistant`

Reply:
actually we have 2 UI part for renaming:
1. For java
![selection_055](https://user-images.githubusercontent.com/1640058/45018222-380c8c80-b032-11e8-90f4-3922412119d4.png)

2. For other languages
![selection_056](https://user-images.githubusercontent.com/1640058/45018232-3f339a80-b032-11e8-99b3-582159609aab.png)

Reply:
I see, thank you for the explanation.